Source/ACE.DatLoader/DatReader.cs

```diff
@@ -93,16 +93,37 @@ public float ReadSingle()
             return data;
         }
 
+        /// <summary>
+        /// Returns a string as defined by the first byte's length
+        /// </summary>
         public string ReadPString()
         {
-            // Returns a string as defined by the first byte's length.
             int stringlength = this.ReadByte();
             byte[] thestring = new byte[stringlength];
             Array.Copy(Buffer, Offset, thestring, 0, stringlength);
             Offset += stringlength;
             return System.Text.Encoding.ASCII.GetString(thestring);
         }
 
+        /// <summary>
+        /// Returns a string as defined by the first byte's length and removes the obfuscation
+        /// </summary>
+        public string ReadOString()

```

Comment:
What's an "OString"?  heck, what's a "PString" for that matter.  I realize you're deobfuscating strings, but let's deobfuscate method names while we're at it.